libs/blocks/figure/figure.js

```diff
@@ -21,6 +23,7 @@ export function buildFigure(blockEl) {
       }
       const video = clone.querySelector('video');
       if (video) {
+        applyHoverPlay(video);
```

Comment:
Since `applyHoverPlay` is only used when there is a video - we should instead dynamically import decorate.js.  `applyHoverPlay` can be applied async so it should not affect perf
```suggestion
        import('../../utils/decorate.js').then(({ applyHoverPlay }) => applyHoverPlay(video));
```

Reply:
done